README.md

```diff
@@ -296,3 +296,147 @@ JSArray args = new JSArray();
 // ...
 connection.event("interfaceName", "methodName", args);
 ```
+
+### JSTP compiler
+
+JSTP compiler is a nice feature for better handling of JSTP packets. You can declare interfaces comfortable for concrete handling purposes and use them instead of `ManualHandler`.
+
+#### Installation
+
+[Check for the latest version](https://bintray.com/metarhia/maven/jstp-compiler)
+
+Gradle:
+```
+dependencies {
+  compile 'com.metarhia.jstp:jstp-compiler:0.1.12'
```

Comment:
I think it's better to use new [gradle syntax](https://docs.gradle.org/current/userguide/artifact_dependencies_tutorial.html) for compile and other methods or add both